x/stake/handler_test.go

```diff
@@ -160,6 +165,32 @@ func TestDuplicatesMsgCreateValidator(t *testing.T) {
 	assert.Equal(t, Description{}, validator.Description)
 }
 
+func TestDuplicatesMsgCreateValidatorOnBehalfOf(t *testing.T) {
```

Comment:
I'd like to see a really simple test which explicitly fails if one or both of the required signers is incorrect

Reply:
Is this necessary? I have a test in msg_test that tests whether GetSigners works correctly. If a tx passes without the signatures from GetSigners, that seems like a problem with ante not with x/stake

Reply:
Yeah, maybe unnecessary - up to you